### ISSUE TITLE
Mathew Lubari.md

### ISSUE THREAD

**Issue by gibertgibs** [ASKnet-Open-Training/ASKnet-Track-II-participants-stories-Community-Empowerment-2020-2021#30]

**Comment by Math-TECH-Hub** [comment]
I am a South Sudanese by in born in Yei in October 16th, 1991 in a small village called Jombu, 
I grew up in South Sudan but most of my time spent in Uganda at school. But in 2016 I came to Uganda as a refugee due to the conflict and I got registered in Bibibidi settlement where I started volunteering with RMF, Action Against Hunger and MSF France – in Uganda as a Community Informant supporting in community mobilization of PSNs and data collection and analysis services before I moved to CTEN where my profession lies.

Technology is my passion, I started it when I was still in senior 3 in 2008 when that was the time I accessed a mobile phone. The funny thing was when my friend used to tease to open the phone lock yet I didn’t know.
I am a highly resourceful, reasonably flexible, innovative, and enthusiastic IT personnel with a Diploma in Information Technology from the University of East Africa. I am a quick learner who can absorb new ideas and is experienced in provision of computer hardware and software support, as well as conducting ICT skills training. I am a well-organized and excellent team player with proven ability to work proactively in a complex and busy office environment. 

Currently I am working for Community Technology Empowerment Network(CTEN) as a Project Manager I started as a volunteering in the ICT department tasked with the role of managing all the innovation centers in Rhino Camp and Bidi Bidi. Before joining CTEN in 2017, I volunteered with RMF, Action Against Hunger and MSF France – in Uganda as a Community Informant supporting in community mobilization of PSNs and data collection and analysis services. 

Services that i do:

1.	Computer networking and system administration
 
2.	Computer repair and maintenance
3.	ICT skills trainings
4.	Fixing of electronic devices
5.	Management skills
